.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node
+node_js:
+- 14.15.1
+addons:
+  ssh_known_hosts:
+  - 134.122.39.75
+branches:
+  only:
+  - main
+  - ci_cd_liwg
+script:
+- openssl aes-256-cbc -K $encrypted_b0d51b9ff91b_key -iv $encrypted_b0d51b9ff91b_iv -in deploy.enc -out deploy -d
+- chmod 600 deploy && mv deploy ~/.ssh/id_rsa
+deploy:
+- provider: script
+  skip_cleanup: true
+  script: sh scripts/server_deploy.sh
+  on:
+    all_branches: true
+    condition: "$TRAVIS_BRANCH =~ ^(main|ci_cd_liwg)$"
```

Comment:
Wasn't `ci_cd_liwg` temporary here too?

Reply:
Yes, It was temporary, but then I gave a thought that if we want to update the ci_cd, we can utilize this branch in future?